server/app.js

```diff
@@ -21,6 +22,7 @@ app.use(bodyParser.json());
 app.use(cors());
 app.use('/', router);
 app.use('/dashboard', dashboard);
+app.use('/board', boardSettings);
```

Comment:
```suggestion
app.use('/settings', boardSettings);
```

Reply:
this is under setting but this is API route and which is under the board. And it will be easy to read

Reply:
Some things not right here 

Either ...
```suggestion
app.use('/board', boardRoutes);
```
Or
```suggestion
app.use('/setting', SettingRoutes);
```


Or maybe your meaning is,...
```suggestion
app.use('/board', boardRoutes);
```
```js
// routes/board.js
router.use('/setting', boardSetting);
```

Reply:
'/board' this route is for all board details. I will move all this route of board settings to the board routes. I have created a temporary route file so that there is no dependency and no conflict occurs. After implementing board routes will move all board settings route to that file,
After that it will be like app.use('/board', board);

app.use('/board', boardRoutes);
--




app.use('/board', boardRoutes);
--

Reply:
Don't do things that are temporary.

What If I want to make API later on and I find no board routes, then I'll make that and this `boardSettings` will be independent of that.

And then you have to refactor that to make sense for settings of the board.


So, don't do things temporary anywhere.

Reply:
Make a proper route for `/board` and add this setting there.

Reply:
I think you are not getting what I want to say, Pravesh already created board route, So to not occurred the conflicts I created another route with name boardSettings. When The code will merge or later I will move all routes of boardSetting to board route

Reply:
Ok.